### DISCUSSION TITLE
OAuth2 webhooks

### DISCUSSION THREAD

**Discussion by cohen2804** [ory/hydra#3779] [Ideas]
I implemented a WEBHOOK to add a claim to a JWT token. 
The flow works, but I need to add a claim based on a value from the client. 
How can I get a value from the client to the WEBHOOK payload (maybe inside extra)? 
I tried to add a URL parameter to the authURL and to the exchange, but it isn't being received by the WEBHOOK.
I am developing with Go. 
[https://www.ory.sh/docs/hydra/guides/claims-at-refresh#webhook-authentication](url)

Thanks

**Comment by terev** [comment]
@cohen2804 Did you actually find a workaround for your use case? If not, I think my pr might address this use case if it gets approved https://github.com/ory/hydra/pull/3790 .
